docs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "doctocat-template",
+  "repository": "primer/doctocat-template",
+  "version": "1.0.0",
+  "scripts": {
+    "clean": "gatsby clean",
+    "develop": "gatsby develop",
+    "build": "npm i && gatsby build"
+  },
```

Comment:
~~Do we need to define a `now-build` script?~~
Oh, just looked at the docs and we don't need to define a `now-build` script to make the now builder work  🎉Very cool